README.md

```diff
@@ -26,46 +26,53 @@ for the purposes of your specific custom content element.
 
 1. Run `npx @tailor-cms/tce-vue` and follow the instructions to setup the project
 
-#### `content-element` folder
+#### `packages` folder
 
 ##### Folder structure
 <ul>
-  <li>index.ts - contains custom element manifest</li>
-  <li>typings.ts - contains TypeScript type definitions</li>
+  <li>content-element subfolder
+    <ul>
+      <li>index.ts - contains custom element manifest</li>
+      <li>types.ts - contains TypeScript type definitions</li>
```

Comment:
Consider renaming this to interfaces and implementing everything as an interface that needs to be implemented.